### PR TITLE
[PT FE] Add aten::log10

### LINKED ISSUE #20581: [Good First Issue]: Support aten::log10 for pytorch models
### Context

OpenVINO component responsible for support of PyTorch models is called as PyTorch Frontend (PT FE). PT FE converts a model represented as TorchScript model to a model in [OpenVINO opset](https://docs.openvino.ai/latest/openvino_docs_ops_opset.html).

### What needs to be done?

- Implement conversion rule and/or transformation to support the new operation.
- Implement operation tests in tests/layer_tests/pytorch_tests. Please consider different data types, but keep reasonable number of test cases

### Example Pull Requests

#18998

### Resources

- [torch.log10 description](https://pytorch.org/docs/stable/generated/torch.log10.html)
- [What is OpenVINO?](https://github.com/openvinotoolkit/openvino#what-is-openvino-toolkit)
- [OpenVINO PyTorch Frontend](https://github.com/openvinotoolkit/openvino/blob/master/src/frontends/pytorch/README.md)
- [Contribution guide](https://github.com/openvinotoolkit/openvino/blob/master/CONTRIBUTING.md)
- [Blog post on contributing to OpenVINO](https://github.com/openvinotoolkit/openvino/blob/master/CONTRIBUTING.md)
- [User documentation](https://docs.openvino.ai/)

### Contact points

@openvinotoolkit/openvino-pytorch-frontend-maintainers 

### Ticket

TBD

### PR DESCRIPTION
**Pull request by siddhant-0707**
### Details:
 - Add support for aten::log10 for pytorch models

### Tickets:
 - Closes #20581

### CODE REVIEW

File: tests/layer_tests/pytorch_tests/test_log.py
Comment (mvafin) [inline tests/layer_tests/pytorch_tests/test_log.py:22]:
```suggestion
            "log10_": torch.log10_,
```

File: tests/layer_tests/pytorch_tests/test_log.py
Comment (mvafin) [inline tests/layer_tests/pytorch_tests/test_log.py:50]: Please also add int case for non-inplace version

**Review by mvafin — APPROVED** [review]
LGTM, let's wait for jenkins results before merge

**Comment by siddhant-0707** [comment]
build_jenkins

**Comment by siddhant-0707** [comment]
build_jenkins

**Comment by mvafin** [comment]
build_jenkins

**Pull request merged**
